elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metric/AggregatedMetric.java

```diff
@@ -71,12 +42,13 @@ public static String functionFormat(Class<? extends Aggregation> aggregationType
      * @param aggregations  A list of all supported aggregations on this {@link Metric}
      */
     public AggregatedMetric(
+            Schema schema,
```

Comment:
Javadoc should be updted to include `@schema`

Reply:
👍